app/src/main/java/com/example/superheroapi/model/SuperHeroes.kt

```diff
@@ -6,15 +6,15 @@ import kotlinx.serialization.Serializable
 @Serializable
 data class SuperHero(
     val response: String? = null,
-    val id: String,
-    val name: String,
+    val id: String = "",
+    val name: String = "",
     @SerialName("powerstats")
     val powerStats: PowerStats? = null,
     val biography: Biography? = null,
     val appearance: Appearance? = null,
     val work: Work? = null,
     val connections: Connections? = null,
-    val image: Image
+    val image: Image = Image("")
 )
 
```

Comment:
Added the default blank parameters so we can show a Preview with an empty SuperHero or only include the fields we want.